### DISCUSSION TITLE
p-dropdown not autoclosing on outside click

### DISCUSSION THREAD

**Discussion by codebyfaizal** [primefaces/community#1090] [PrimeVue]
I am using primevue with vuetify and p-dropdown panel is not auto closing on outside click any solution?

<img width="435" alt="Screenshot 2024-02-02 at 12 24 04 PM" src="https://github.com/primefaces/community/assets/129145958/c3e1eccb-b210-4161-87a0-3bd4f34fdb36">

**Comment by tugcekucukoglu** [comment]
Cannot replicate the problem? Do you have a stackblitz link?
